### PR TITLE
Add a command-line interface

### PR DESCRIPTION
**Pull request by mara004**

### CODE REVIEW

File: src/pypdfium2/__main__.py [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+def main():
+    
+    args = parse_args()
+    
+    if args.version:
+        print(f"PyPDFium2 {pdfium.__version__}", f"PDFium {pdfium.__pdfium_version__}", sep='\n')
+        sys.exit()
+    
+    assert args.pdffile is not None
```
Comment (adam-huganir) [inline src/pypdfium2/__main__.py]: `-O` as a python argument will disable assert statements, so we should make this into an explicit `ArgumentError`

File: src/pypdfium2/__main__.py [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+from os.path import (
+    join,
+    basename,
+    splitext
+)
+
+
+def rotation_type(string):
+    rotation = int(string)
+    assert rotation in (0, 90, 180, 270)
```
Comment (adam-huganir) [inline src/pypdfium2/__main__.py]: assert -> raise ValueError

File: src/pypdfium2/__main__.py (hunk 1 above)
Comment (mara004) [reply src/pypdfium2/__main__.py]: I've changed this to ValueError - didn't figure out how to use `argparse.ArgumentError`.

File: src/pypdfium2/__main__.py (hunk 2 above)
Comment (mara004) [reply src/pypdfium2/__main__.py]: done

**Pull request merged**